rpc_to_mongo_db.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import argparse
+import base64
+import pymongo
+import requests
+import yaml
+import json
+import hashlib
+from Crypto.Cipher import AES
+
+
+def get_collection(cfg):
+    client = pymongo.MongoClient(cfg["database"]["connection_string"])
+    database_name = cfg["database"]["database_name"]
+    collection_name = cfg["database"]["collection_name"]
+    return client[database_name][collection_name]
+
+
+def get_config():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("config_filepath", type=str, help="path_to_config_file")
+    arguments = parser.parse_args()
+    with open(arguments.config_filepath, "r") as ymlfile:
+        cfg = yaml.safe_load(ymlfile)
+        return cfg
+
+
+def get_attr_perun_names(cfg):
+    names = []
+    for attribute in cfg["attributes"].values():
+        if isinstance(attribute, list):
+            for val in attribute:
+                names.append(val)
+        else:
+            names.append(attribute)
+    return names
+
+
+def get_facility_attributes_url(cfg, facility_id):
+    url = cfg["rpc"]["perunUrl"] + "attributesManager/getAttributes?facility=" + str(facility_id)
+    for attribute_name in get_attr_perun_names(cfg):
+        url += "&attrNames[]=" + attribute_name
+    return url
+
+
+def get_data(cfg):
+    rpc_url = cfg["rpc"]["perun_url"]
+    rpc_facilities = rpc_url + "searcherManager/getFacilities?attributesWithSearchingValues{}=" + \
+        cfg["attributes"]["rp_type"] + "=" + cfg["rp_type_attr_value"] + "&attributesWithSearchingValues{}=" + \
+        cfg["attributes"]["proxy_identifier"] + "=" + cfg["proxy_identifier_value"]
```

Comment:
I think the correct format for passing named keys is `url?array[key1]=value1&array[key2]=value2`, therefore:
```suggestion
    rpc_facilities = "{}searcherManager/getFacilities?attributesWithSearchingValues[{}]={}&attributesWithSearchingValues[{}]={}".format(rpc_url, cfg["attributes"]["rp_type"], cfg["rp_type_attr_value"], cfg["attributes"]["proxy_identifier"], cfg["proxy_identifier_value"])
```